packages/datadog_flutter_plugin/lib/src/datadog_sdk_method_channel.dart

```diff
@@ -8,13 +8,29 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
 import '../datadog_flutter_plugin.dart';
+import 'android/android_plugin_stub.dart'
+    if (dart.library.io) 'android/android_plugin_ffi.dart';
 import 'datadog_sdk_platform_interface.dart';
 import 'internal_logger.dart';
 
 class DatadogSdkMethodChannel extends DatadogSdkPlatform {
   @visibleForTesting
   final methodChannel = const MethodChannel('datadog_sdk_flutter');
 
+  LogCallback? _logCallback;
+
+  // Used by iOS, not by Andoid at the moment
+  DatadogContext? _cachedContext;
+  @override
+  DatadogContext? get cachedContext {
+    if (Platform.isIOS) {
+      return _cachedContext;
+    } else if (Platform.isAndroid) {
+      return AndroidDatadogFlutterPlugin.getContext();
```

Comment:
we can probably also cache and invalidate it when `setUser`/`setAccount`/`addExtraInfo` calls are made

Reply:
No, I'm talking about the Flutter methods. Android won't cache anything here, because `DatadogContext` includes many properties.